docs/uart.rst

```diff
@@ -35,6 +35,11 @@ Functions
         * 57600
         * 115200
 
+    .. note::
```

Comment:
Could we move this note to the read method?
It'd be nice to include as well the calculation (unit is milliseconds) `microbit_uart_timeout_char = 13000 / baudrate + 1`